Core/HLE/sceFont.cpp

```diff
@@ -1171,6 +1175,11 @@ static int sceFontFindOptimumFont(u32 libHandle, u32 fontStylePtr, u32 errorCode
 	if (nearestFont) {
 		optimumFont = nearestFont;
 	}
+	//Special chinese font support
+	if ((g_Config.iLanguage == 11) || (g_Config.iLanguage == 10)) {
+		if (optimumFont == internalFonts[2])
+			optimumFont = internalFonts[0];
```

Comment:
If "zh_gb.pgf" exits, use "zh_gb.pgf" instead of "chn.pgf"?

Reply:
if (optimumFont == internalFonts[2]) should be return false

Reply:
But if "zh_gb.pgf" exits, this index would be wrong.